app/views/wishlist_items/_wishlist_item.html.erb

```diff
@@ -23,15 +23,15 @@
 
         <div class="row">
           <!-- Staff Message -->
-          <% if wishlist_item.staff_message %>
-            <p>
-            <strong>Staff Message:</strong>
-            <span class="card-text"><%= wishlist_item.staff_message %></span>
+          <% if wishlist_item.staff_message && wishlist_item.staff_message.length > 0 %>
```

Comment:
You can just do `.present?` instead of length checking.

Reply:
Ah thank you.